lib/mix/tasks/phoenix.digest.ex

```diff
@@ -35,13 +36,14 @@ defmodule Mix.Tasks.Phoenix.Digest do
   """
 
   def run(args) do
-    {opts, args, _} = OptionParser.parse(args, aliases: [o: :output])
+    {opts, args, _} = OptionParser.parse(args, aliases: [o: :output], switches: [version_param: :boolean])
     input_path  = List.first(args) || @default_input_path
     output_path = opts[:output] || input_path
+    version_param = opts[:version_param] || @version_param
```

Comment:
This means version_param will never be false because it will always be `anything || true`. We need to make the tests fail and fix this accordingly by using `Keyword.get/3`.